indy-besu/smart_contracts/test/did/DidRegistry.spec.ts

```diff
@@ -39,11 +48,12 @@ describe('DIDContract', function () {
     })
 
     it('Should fail if the DID being created already exists', async function () {
-      const { didRegistry } = await loadFixture(deployDidContractFixture)
+      const { didRegistry, testAccounts } = await loadFixture(deployDidContractFixture)
 
       const did: string = 'did:indy2:testnet:SEp33q43PsdP7nDATyySSH'
       const didDocument = createBaseDidDocument(did)
 
+      didRegistry.connect(testAccounts.trustee.account)
```

Comment:
Since the `didRegistry.connect(testAccounts.trustee.account)` operation is executed in most tests, maybe it's  better to move it to the `deployDidContractFixture` method?

Reply:
`loadFixture` is executed once, takes a snapshot and later it is not executed again in each test, but returns to the snapshot and returns `didRegistry` instance

Reply:
Since `deployDidContractFixture` executed only once, can we replace it with `beforeAll` hook and do `didRegistry.connect(testAccounts.trustee.account)`  in beforeEach?
Or just do both in beforeEach?